html/package.json

```diff
@@ -7,22 +7,24 @@
     "lib": "lib"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "clean": "rm -rf assets output ilib.js ilib-include.js webpack.config.js",
     "scan": "ilib-scanner --assembly=assembled --locales=ko-KR,es-ES ilib-include.js",
-    "build": "webpack"
+    "build": "webpack",
+    "gen-res-assemble": "ilib-resbundler --assembly assembled --compiled compiled --resDir resources --outDir outResDir",
+    "gen-res-dynamic": "ilib-resbundler --assembly dynamic --compiled compiled --resDir resources --outDir outResDir --locales 'ko-KR,es-ES'",
+    "clean": "rm -rf assets output ilib.js ilib-include.js webpack.config.js outResDir"
   },
   "author": {
     "name": "Goun Lee",
     "email": "goun.lee@lge.com"
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "ilib-scanner": "^1.3.0",
+    "ilib-resbundler": "^1.0.1",
+    "ilib-scanner": "^1.3.1",
     "ilib-webpack-loader": "^1.3.1",
     "webpack": "^3.10.0"
   },
   "devDependencies": {
-    "webpack-cli": "^4.9.1"
+    "webpack-cli": "^4.9.2"
```

Comment:
Add loctool to the dev dependencies?

Reply:
This package should consider the case not use loctool, 
so I wanted to avoid loctool dependency in this sample, because currently webOS TV uses a Java localization tool, and might need this sample guide.
I may generate another sample by using loctool.